x-pack/plugins/maps/public/actions/layer_actions.ts

```diff
@@ -50,9 +50,9 @@ export function trackCurrentLayerState(layerId: string) {
 }
 
 export function rollbackToTrackedLayerStateForSelectedLayer() {
-  return async (dispatch: Dispatch, getState: () => MapStoreState) => {
+  return (dispatch: Dispatch, getState: () => MapStoreState) => {
     const layerId = getSelectedLayerId(getState());
-    await dispatch({
```

Comment:
This is the root of the async clean-up. Dispatching a pure action is sync. So need to wait for anything. The other removals of await and async were anything that called this redux thunk